scripts/stanford-filter.script

```diff
@@ -108,6 +108,78 @@
+
+   // Siemens SOMATOM DEFINITION AS+, software version SYNGO CT VA48A, different resolution
+  +( Manufacturer.startsWithIgnoreCase("SIEMENS") 
+    * Modality.equalsIgnoreCase("CT")
+    * ImageType.contains("DERIVED")
+    * ManufacturerModelName.containsIgnoreCase("SOMATOM DEFINITION AS+") 
+    * SecondaryCaptureDeviceManufacturerModelName.containsIgnoreCase("")
+    * [0018,1020].containsIgnoreCase("SYNGO CT VA48A")
+    * BurnedInAnnotation.equalsIgnoreCase("")
+    * DerivationDescription.equalsIgnoreCase("")
+    * Rows.equals("519") * Columns.equals("512")
+        + Rows.equals("526") * Columns.equals("512")
+        + Rows.equals("587") * Columns.equals("512")
+        + Rows.equals("613") * Columns.equals("512")
+        + Rows.equals("633") * Columns.equals("512")
+        + Rows.equals("665") * Columns.equals("512")
+        + Rows.equals("679") * Columns.equals("512")
+        + Rows.equals("692") * Columns.equals("512")
+        + Rows.equals("722") * Columns.equals("512")
+        + Rows.equals("775") * Columns.equals("512")
+        + Rows.equals("868") * Columns.equals("512")
+        + Rows.equals("512") * Columns.equals("512")
+        + Rows.equals("512") * Columns.equals("532")
+        + Rows.equals("512") * Columns.equals("535")
+        + Rows.equals("512") * Columns.equals("547")
+        + Rows.equals("512") * Columns.equals("565")
+        + Rows.equals("512") * Columns.equals("605")
+        + Rows.equals("512") * Columns.equals("630")
+        + Rows.equals("512") * Columns.equals("632")
+        + Rows.equals("512") * Columns.equals("647")
+        + Rows.equals("512") * Columns.equals("653")
+        + Rows.equals("512") * Columns.equals("682")
+        + Rows.equals("512") * Columns.equals("708")
+        + Rows.equals("512") * Columns.equals("753")
+   )
+
+   // Siemens SOMATOM FORCE, different resolution
+
+   // Siemens SOMATOM DEFINITION FLASH, version SYNGO CT VB20A, SCRUBBED
+  +( Manufacturer.startsWithIgnoreCase("SIEMENS") 
+   * Modality.equalsIgnoreCase("CT")
+   * ImageType.contains("DERIVED")
+   * ManufacturerModelName.containsIgnoreCase("SOMATOM DEFINITION FLASH") 
+   * [0018,1020].containsIgnoreCase("syngo CT VB20A")
+   * ([0018,1018].containsIgnoreCase("Volume Viewer")+[0018,1018].containsIgnoreCase(""))
+   )
+
+   // Siemens secondary_capture_device_manufacturer AQNET
+  +(Manufacturer.startsWithIgnoreCase("SIEMENS") * (Modality.equalsIgnoreCase("CT") + Modality.equals("PT"))
+   *(ImageType.contains("DERIVED"))
+   *(ImageType.contains("AQNET"))
+   *[0018,1018].containsIgnoreCase("AQNET")
+   )
+
+   // Siemens Healthineers manufacturers SYNGO.VIA.VB30A, 
+  +(Manufacturer.startsWithIgnoreCase("Siemens Healthineers") * (Modality.equalsIgnoreCase("CT") + Modality.equals("PT"))
+   *(!BurnedInAnnotation.equalsIgnoreCase("YES"))
+   *(ImageType.contains("DERIVED"))
+   *
+    (ManufacturerModelName.containsIgnoreCase("SYNGO.VIA.VB30A"))
+    * [0018,1020].containsIgnoreCase("")
```

Comment:
If this rule is created based on the query result, the query returned "", but actually this tag might be missing in the series metadata. If so, this rule will not work for such series.